python/pyspark/sql/connect/client/core.py

```diff
@@ -15,7 +15,8 @@
 # limitations under the License.
 #
 __all__ = [
-    "ChannelBuilder",
+    "DefaultChannelBuilder",
+    "DefaultChannelBuilder",
```

Comment:
remove one?

Reply:
Should have been "ChannelBuilder", "DefaultChannelBuilder", thanks!